scripts/upgrade

```diff
@@ -126,7 +128,7 @@ then
   tmpdir="$(mktemp -d)"
 
   # Backup the config file and local plugins in the temp dir
-  cp -a "$final_path/config/config.json" "$tmpdir/config.json"
+  #cp -a "$final_path/config/config.json" "$tmpdir/config.json"
```

Comment:
I guess we would need to selectively patch the local config.json file if the the postgres migration is executed.

Reply:
Done